app/generate_figures.py

```diff
@@ -1,13 +1,15 @@
-from input import input_path
-from utilities import landuse
 from figures import *
 import os
 import pandas as pd
+from config import Config
+from land_use import LandUse
 pd.set_option('mode.chained_assignment', None)
 
 
 def generate_figures(url):
-    results_path = input_path(url)
+    config = Config(url)
+    land_use = LandUse(config)
```

Comment:
This is where the conversion to classes shine:
- We create a `Config` object that knows which model we are running
- Then we can pass the `config` object to the `LandUse` class that will know where to get the data from there.